assets/data-port/import/data/reducer.js

```diff
@@ -90,7 +90,7 @@ export default ( state = DEFAULT_STATE, action ) => {
 		case ERROR_FETCH_IMPORT_DATA:
 			// No need to start a new job until we have our first active upload.
 			const isErrorNoActiveJob =
-				action.error.code === 'sensei_data_port_no_active_job';
+				action.error.code === 'sensei_data_port_job_not_found';
```

Comment:
Could this cause issues when querying with an actual job id instead of `active`, that was removed for some reason? Though this seems to be only used for the initial active query now, maybe just adjusting naming along the lines of `FETCH_ACTIVE_JOB` would clear it up.
Or having a separate endpoint to get the active job that just returns empty instead of an error?

Reply:
With b294057081da8e38dcc9dd1f5843e51aaeb9b655 this is now only used for `active`.